### PR TITLE
Adds Stand and fixes ped button

### PR DESCRIPTION
**Pull request by GamearooCodes**
Adds the stand to the mod
image

### CODE REVIEW

**Comment by GamearooCodes** [comment]
the bulbs and gui changes where removed

**Comment by CyberedCake** [comment]
Oooh, very cool!

**Comment by GamearooCodes** [comment]
Fixed ped button when angled to render correctly 
image < broken
image < fixed

File: src/main/java/com/clussmanproductions/trafficcontrol/blocks/BlockStand.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+public class BlockStand extends Block {
+	public static PropertyInteger ROTATION = PropertyInteger.create("rotation", 0, 15);
+	public BlockStand()
+	{
+		super(Material.IRON);
+		setRegistryName("stand");
+		setUnlocalizedName(ModTrafficControl.MODID + ".stand");
+		setCreativeTab(ModTrafficControl.CREATIVE_TAB);
+		setLightOpacity(1);
```
Comment (CSX8600) [inline src/main/java/com/clussmanproductions/trafficcontrol/blocks/BlockStand.java:30]: Probably should add some kind of hardness values and harvest level and stuff. See other blocks for examples (as long as they have them, I know I'm missing that in a couple spots)

**Review by CSX8600 — CHANGES_REQUESTED** [review]
After some consideration, I don't want to overload things by adding 39 different bulbs. Instead, I would like to eventually add custom bulb support where it would show up in manual mode/OC. For auto mode, custom bulbs would have an "equivalent" bulb that they would show up alongside with.

With that said, I think the U-Turn bulbs should wait until that feature comes (probably in like 7 years LOL) and should be pulled out of this PR.

Will do another round of review on this PR afterwards.

**Comment by GamearooCodes** [comment]
> After some consideration, I don't want to overload things by adding 39 different bulbs. Instead, I would like to eventually add custom bulb support where it would show up in manual mode/OC. For auto mode, custom bulbs would have an "equivalent" bulb that they would show up alongside with.
> 
> With that said, I think the U-Turn bulbs should wait until that feature comes (probably in like 7 years LOL) and should be pulled out of this PR.
> 
> Will do another round of review on this PR afterwards.

if i have to pull the uturn bulbs its best to just close this pr as theres like 20+ files i would have to remove and i have no way of testing it at this time if i do it was a pain last time i had to remove frames and the bulbs with the gui changes can break the gui if that changes anyway 

Did the edits as it wasnt hard thanks to github

File: src/main/java/com/clussmanproductions/trafficcontrol/blocks/BlockStand.java (hunk 1 above)
Comment (GamearooCodes) [reply src/main/java/com/clussmanproductions/trafficcontrol/blocks/BlockStand.java:30]: this is done on purpuse as its ment to be easily moved as a sign stand would be irl

**Comment by GamearooCodes** [comment]
i pushed some reverts removing the changes to the control box and uturn bulbs leaving just the stand and the ped button fix


also the hardness is ment to not be there as to be movable by hand like a irl foldable stand

All tested and works

**Review by CSX8600 — APPROVED** [review]

**Pull request merged**